api/src/routes/api.ts

```diff
@@ -6,11 +6,13 @@ import Config from "../models/Config";
 import Job from "../models/Job";
 
 const router = express.Router();
+const { body, validationResult } = require("express-validator");
```

Comment:
I don't think this is getting used at all.

Reply:
Fixed!